RLTest/redis_std.py

```diff
@@ -195,12 +196,20 @@ def stopEnv(self):
             self.slaveProcess = None
         self.envIsUp = False
 
-    def getConnection(self):
-        return redis.Redis('localhost', self.port, password=self.password)
+    def getConnection(self, decode_responses=True):
+        return redis.Redis('localhost', self.port, password=self.password, decode_responses=decode_responses)
```

Comment:
@lantiga the issue is that decode_responses is not working with Python2.7, I still want this framework to work on Python2.7.
So I suggest whenever we creating a redis client, i.e, redis.Redis( ... ), we will check if the decode_responses=True and if it does we will give it as an argument to the function, otherwise we will not give it at all.
Then I think that when we start the test we check the python version and decide how to set the decode_responses default value.
What do you think?

Reply:
Great point @MeirShpilraien, I tried to update the PR with a better behavior for `decode_responses` in python 2 vs 3.